package.json

```diff
@@ -64,10 +68,13 @@
   },
   "scripts": {
     "build": "tsc",
+    "initdb": "ts-node ./src/tools/initdb",
     "lint": "eslint . --ext .js,.ts",
     "lint:fix": "eslint . --fix --ext .js,.ts",
     "start": "node ./build/index.js",
     "start:dev": "nodemon --exec 'node --require ts-node/register ./src/index.ts' --ext .js,.json,.ts",
-    "test": "jest --silent"
+    "test": "jest --silent",
+    "test:unit": "jest --silent --testNamePattern '^((?!#integration).)*$'",
```

Comment:
Since we are running all tests in CI we should not see any skipped tests there. The downside of running all tests in CI is that if we do not tag our integration tests with `#integration` in the describe name then it will still pass in CI.